proposal.rst

```diff
@@ -239,7 +308,64 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
+**Month 1 Theme** : *DOCUMENTATION AND OMEGA *
+
+**Week 1** 
+	- Read and understand code-base further. 
+	- Identify the families of algorithms inside the code. Currently known to me are weighting-scheme, letor, spelling-checking, cluster, stemming
+
+**Week 2** 
+	- solve "Rework Omega templates to use more modern web techniques"
```

Comment:
Above you said you plan to do this before the GSoC results are out.

Reply:
Yes. I will start tackling, probably starting this weekend. However, still I wrote that in Initial week schedule in case some delay happens.